packaging/fedora/snapd.spec

```diff
@@ -320,11 +331,19 @@ export GOPATH=$(pwd):%{gopath}
 export GOPATH=$(pwd):$(pwd)/Godeps/_workspace:%{gopath}
 %endif
 
-%gobuild -o bin/snap %{import_path}/cmd/snap
-%gobuild -o bin/snap-exec %{import_path}/cmd/snap-exec
-%gobuild -o bin/snapctl %{import_path}/cmd/snapctl
-%gobuild -o bin/snapd %{import_path}/cmd/snapd
-%gobuild -o bin/snap-update-ns %{import_path}/cmd/snap-update-ns
+GOFLAGS=
```

Comment:
Who else uses GOFLAGS? Nothing in the spec file does and nothing of the RPM golang macros does.